.circleci/config.yml

```diff
@@ -1278,12 +1314,11 @@ workflows:
             - run-snowplow-micro
             - wait-for-databases
 
-      - fe-tests-cypress:
+      - fe-tests-cypress-visual:
           name: percy-visual-tests-<< matrix.edition >>
           requires:
             - build-uberjar-<< matrix.edition >>
           cypress-group: "percy-visual-<< matrix.edition >>"
-          test-files: "./frontend/test/metabase-visual/**/*.cy.spec.js"
```

Comment:
Looking at the CircleCI logs I can see that visual tests ran correctly, but I'm trying to understand how? Asking because you removed the pointer to the source files which should run on this line.

Reply:
Because now we are running `test-visual-no-build` and the files config are already on the [package.json](https://github.com/metabase/metabase/blob/aaf1b6011116a3b5dd390d96595b64db0abf9111/package.json#L302)

Reply:
But we already have [`percy.yml`](https://github.com/metabase/metabase/blob/master/.github/workflows/percy.yml#L62) for that.

`test-visual-no-build` will **actually record** Percy screenshots (note the `percy exec` part) and again - it will do that for every single commit on every branch. Running percy on CirceCi is not meant to record anything. It's merely a sanity check that we don't break anything.